samples/deoptim/deoptim.r

```diff
@@ -0,0 +1,69 @@
+# =============
+# === Setup ===
+# =============
+
+# install packages
+install.packages("DEoptim")
+library(DEoptim)
+
+library(devtools)
+install_github("azure/razurebatch")
+install_github("azure/doazureparallel")
+
+# import the doAzureParallel library and its dependencies
+library(doAzureParallel)
+
+credentialsFileName <- "credentials.json"
+clusterFileName <- "cluster.json"
+
+# generate a credentials json file
+generateCredentialsConfig(credentialsFileName)
+
+# set your credentials
+setCredentials(credentialsFileName)
+
+# generate a cluster config file
+generateClusterConfig(clusterFileName)
+
+# Create your cluster if not exist
+cluster <- makeCluster(clusterFileName)
+
+# register your parallel backend
+registerDoAzureParallel(cluster)
+
+# check that your workers are up
+getDoParWorkers()
+
+# ==============================================================
+# === Caling DEoptim with doAzureParallel cluster as backend ===
+# ==============================================================
+
+## The below examples shows how the call to DEoptim can be parallelized
+## with doAzureParallel cluster as backend
+Genrose <- function(x) {
+  ## One generalization of the Rosenbrock banana valley function (n parameters)
+  n <- length(x)
+  ## simulate a long running task ...
+  Sys.sleep(60)
```

Comment:
Are there any other samples for running a long task?

Reply:
This sample looks promising? https://mintegration.eu/category/deoptim/? We should try running some benchmarks on this sample